DQM/SiPixelPhase1TrackClusters/src/SiPixelPhase1TrackClusters.cc

```diff
@@ -142,6 +161,7 @@ void SiPixelPhase1TrackClusters::analyze(const edm::Event& iEvent, const edm::Ev
         histo[ONTRACK_SIZE      ].fill(double(cluster.size()  ), id, &iEvent);
         histo[ONTRACK_POSITION_B].fill(clustgp.z(),   clustgp.phi(),   id, &iEvent);
         histo[ONTRACK_POSITION_F].fill(clustgp.x(),   clustgp.y(),     id, &iEvent);
+	histo[ONTRACK_SIZE_VS_ETA].fill(etatk[key], cluster.sizeY(), id, &iEvent);
```

Comment:
strickly speaking one should take into account bigpixel for instance as
     auto sizeY = cluster.sizeY();
     if (topol.containsBigPixelInY(cluster.minPixelCol(), cluster.maxPixelCol()) ) sizeY+=1;
we need to ask confirmation to Pixels guys....